zipline/finance/cancel_policy.py

```diff
@@ -0,0 +1,70 @@
+import abc
+import logbook
+
+from six import with_metaclass, iteritems
+
+from zipline.gens.sim_engine import DAY_END
+from zipline.utils.serialization_utils import (
+    VERSION_LABEL
+)
+
+warning_logger = logbook.Logger('AlgoWarning')
+
+
+def log_warning(amount, sid, filled, policy_name):
+    warning_logger.warn(
```

Comment:
When do we want to use a logger for warnings versus the stdlib `warnings` module?